src/components/multi_select.js

```diff
@@ -104,7 +104,12 @@ export class MultiSelect extends PureComponent {
       searchValue,
       withGrouping,
       generateClassName,
-      listRenderer
+      listRenderer,
+      showDestinationSearch,
+      searchDestinationChanged,
+      searchDestinationValue,
+      filterDestinationItems,
+      filteredDestinationItems
```

Comment:
maybe it's better to use SelectedItems instead of Destination
filteredSelectedItems
showSelectedItemsSearch
...

it's longer but on the other hand none of the input properties use "destination"